Formula/t/tart.rb

```diff
@@ -14,7 +14,10 @@ class Tart < Formula
   end
 
   # https://tart.run/blog/2023/02/11/changing-tart-license/
-  deprecate! date: "2024-09-16", because: "switched to a DFSG-incompatible license"
+  # Original deprecation date: 2024-09-16
```

Comment:
Recovering `deprecate!` is also considerable. It is used with many cask to keep the history, but I am not sure for formula.